build.sbt

```diff
@@ -368,6 +366,19 @@ lazy val codegenPlugin = (projectMatrix in file("modules/codegen-plugin"))
     scriptedBufferLog := false
   )
 
+lazy val cli = (projectMatrix in file("modules/cli"))
```

Comment:
After consideration, I'd like this to be renamed to `decline` (as well as jvm package in which the code is located)

Reply:
Should we rename the actual Scala code or stick with the Smithy4sCli convention

Reply:
that's what I mean by the "jvm package in which the code is located"